pxr/imaging/hdSt/materialXFilter.cpp

```diff
@@ -1453,6 +1429,13 @@ HdSt_ApplyMaterialXFilter(
 
     TRACE_FUNCTION_SCOPE("ApplyMaterialXFilter: Found Mtlx Node.")
 
+    HdSt_MaterialXShaderRegistry* const materialXShaderRegistry =
+        resourceRegistry->GetMaterialXShaderRegistry();
```

Comment:
This is a specialized instance registry that supports a persistent cache. To minimize include dependencies, I expose it to clients of `HdStResourceRegistry` by pointer.